Scripts/SaneDocReport/SaneDocReport.yml

```diff
@@ -2,27 +2,39 @@ args:
 - default: true
   description: sane report json file contents base64
   isArray: false
-  name: "sane_docx_report_base64"
+  name: sane_docx_report_base64
   required: true
   secret: false
+- default: false
+  description: orientation of the report
+  isArray: false
+  name: orientation
+  required: false
+  secret: false
+- default: false
+  description: the paper size of the report
+  isArray: false
+  name: paperSize
+  required: false
+  secret: false
 comment: Parse Sane-json-reports and export them as docx files (used internally).
 commonfields:
   id: SaneDocReports
   version: -1
 enabled: true
 name: SaneDocReports
-script: ''
-fromversion: 5.0.0
+script: '-'
 system: false
 tags:
 - sane-doc-reports
 - sane-reports
 - docx
 timeout: '0'
 type: python
-dockerimage: demisto/sane-doc-reports:1.0.0.1087
+dockerimage: demisto/sane-doc-reports:1.0.0.1443
 runas: DBotWeakRole
 runonce: false
-deprecated: true
 tests:
-- No Test
\ No newline at end of file
+- No Test
+deprecated: true
```

Comment:
why it should be deprecated ?

Reply:
We already talked about this, we didn't want it to be shown as it is an internal automation.